.github/workflows/build.yml

```diff
@@ -5,8 +5,23 @@ on:
   pull_request:
 jobs:
   validation:
-    name: "Gradle Wrapper Validation"
+    name: "Build"
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: gradle/wrapper-validation-action@v1
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Validate Gradle wrapper
+        uses: gradle/wrapper-validation-action@v1
+      - name: Install Java 17
+        uses: actions/setup-java@v3.9.0
```

Comment:
No need to use ".9.0" in setup-java context.

Reply:
Fixed